etna/models/deadline_ma.py

```diff
@@ -200,7 +248,14 @@ def _forecast(self, df: pd.DataFrame, prediction_size: int) -> pd.DataFrame:
         df = df.iloc[-prediction_size:]
         y_pred = result_values[-prediction_size:]
         df.loc[:, pd.IndexSlice[:, "target"]] = y_pred
-        return df
+
+        components = None
```

Comment:
Let's rename it everywhere components, target_components  -> `target_components_df`